### ISSUE TITLE
Installation in a fresh Ubuntu 20.04 is broken (numba related, fix provided)

### ISSUE THREAD

**Issue by sgongora2501** [3fon3fonov/exostriker#100]
A new installation of ExoStriker (commit number 4f3404fd0c7e3b3f2fea9c7274ce0ecba8668010) through 

`python setup.py install --user`

or

`sudo python setup.py install`

throws the following error message when ExoStiker is called using `exostriker`

```
Your dynesty is version<=1.1, switching to the internally imported github version==1.2!
using Dynesty = 1.2 beta
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe

Abortado (`core' generado)
```

**Comment by 3fon3fonov** [comment]
We recently had this problem in Sofia,  and I am quite confident this is a numpy/numba problem! Report your numpy version, please.

You likely need to install the newest numpy version, but then, as far as I remember "wotan" will not work. The fix was to downgrade numba to 0.63

i.e. Try:

```
sudo pip3 install numpy -U
sudo pip3 uninstall numba
sudo pip3 install numba==0.63

```

If this works, then I must make sure the ES works with the new version of numpy (new Ubuntu install).

Let me know.

**Comment by 3fon3fonov** [comment]
I am closing this, since the problem does not seems to exist on Unubtu 22.04+ and Python3.8 and 3.10 (tested).
